dev-tools/validation/check_dev_demo_summary.py

```
@@ -0,0 +1,998 @@
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *_args):
+        return False
+
+    def read(self):
+        return b'{"status":"SUCCESS"}'
+
+
+def _validate_smoke_account(root: Path) -> None:
```

Ruff (0.16.0)

[warning] Too many statements (88 > 50)

(PLR0915)